install_sct

```diff
@@ -113,13 +115,13 @@ function finish() {
   cd "$SCT_SOURCE"
   if [[ "$value" -eq 0 ]]; then
     print info "Installation finished successfully!"
+    rm -r install_sct_log.txt  # if there's no errors, there's no need to keep this log around
```

Comment:
Is this a good idea? Maybe we should just leave it around in all cases, that's a little more consistent.

Reply:
i don't see it as a problem to leave it around indeed. More pros than cons